test/e2e/framework/ginkgowrapper.go

```diff
@@ -353,6 +358,16 @@ func withFeature(name Feature) interface{} {
 // [k8s.io/apiserver/pkg/util/feature.DefaultMutableFeatureGate]. Once a
 // feature gate gets removed from there, the WithFeatureGate calls using it
 // also need to be removed.
+//
+// [Alpha] resp. [Beta] get added to the test name automatically depending
+// on the current stability level of the feature. Feature:Alpha resp.
+// Feature:Beta get added to the Ginkgo labels because this is a special
+// requirement for how the cluster needs to be configured.
```

Comment:
Beta is an edge case here: if the feature gate is on by default, any cluster *might* work out-of-the-box. I am saying "might" because:
- the cluster might explicitly turn off beta features
- the feature might depend on a beta API group - those are not on by default

I think it is safer to assume that "beta" does not mean "available by default" and thus add the `Feature:Beta` label.

Reply:
cc @carlory 

See also https://github.com/kubernetes/test-infra/pull/31665